Sources/VLCSettingsTableHeaderView.swift

```diff
@@ -0,0 +1,65 @@
+/*****************************************************************************
+ * VLCSettingsTableHeaderView.swift
+ * VLC for iOS
+ *****************************************************************************
+ * Copyright (c) 2018 VideoLAN. All rights reserved.
+ * $Id$
+ *
+ * Authors: Mike JS. Choi <mkchoi212 # icloud.com>
+ *
+ * Refer to the COPYING file of the official project for license.
+ *****************************************************************************/
+
+import UIKit
+
+class VLCSettingsTableHeaderView: UITableViewHeaderFooterView {
+    @objc static let identifier = String(describing: VLCSettingsTableHeaderView.self)
+    @objc var headerLabel: UILabel!
+    
+    override var frame: CGRect {
+        get {
+            return super.frame
+        }
+        set {
+            if newValue.width == 0 {
+                return
+            }
+            super.frame = newValue
+        }
+    }
+    
+    override init(reuseIdentifier: String?) {
+        super.init(reuseIdentifier: reuseIdentifier)
+
+        let separator = UIView()
+        separator.backgroundColor = PresentationTheme.current.colors.settingsSeparatorColor
+        separator.translatesAutoresizingMaskIntoConstraints = false
+        contentView.addSubview(separator)
+        
+        let margins = contentView.layoutMarginsGuide
+        let offset = textLabel!.frame.origin.x
+        NSLayoutConstraint.activate([
+            separator.leadingAnchor.constraint(equalTo: margins.leadingAnchor, constant: offset),
+            separator.trailingAnchor.constraint(equalTo: margins.trailingAnchor, constant: offset),
+            separator.heightAnchor.constraint(equalToConstant: 2),
```

Comment:
separators usually have a height of 1 or 0.5 even why the 2?

Reply:
Figma mockups say 2 but changing it to 1 looks a lot better